quickstarts/microsoft.mobilenetwork/mobilenetwork-provision-proxy-sims/azuredeploy.parameters.json

```diff
@@ -19,7 +19,8 @@
                     "internationalMobileSubscriberIdentity": "000000000000001",
                     "authenticationKey": "00112233445566778899AABBCCDDEEFF",
                     "operatorKeyCode": "00000000000000000000000000000001",
-                    "deviceType": "None"
+                    "deviceType": "None",
+                    "simPolicyName": "sim-policy-1"
```

Comment:
The sim policy name in the prerequisites is Default-policy and we aren't allowing the user to override so we should use the same name here.

Reply:
On top of that, there's no need to put the sim policy name in the sim parameters if it's the same for all sims.

I would not make the sim policy name a parameter at all since you must use Default-Policy

Reply:
Addressed.
Moved the SIM Policy attribute to root level, not at SIM level.